src/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+<head>
+    <title>Idea Town</title>
+    <!-- inject:css -->
```

Comment:
@meandavejustice It looks like the injection is happening in [/gulpfile.js#L80](https://github.com/mozilla/idea-town-app/pull/11/files#diff-b9e12334e9eafd8341a6107dd98510c9R80) which is giving me this magic locally (when running `$ gulp server` w/ dev resources):

```

    <!-- inject:css -->
    <link rel="stylesheet" href="/styles/main.css">
    <link rel="stylesheet" href="/styles/vendor/normalize.css">
    <link rel="stylesheet" href="/styles/vendor/normalize.css/normalize.css">
    <link rel="stylesheet" href="/styles/vendor/mozilla-tabzilla/css/tabzilla.css">
    <!-- endinject -->

...

  <!-- inject:js -->
  <script src="/app/app.js"></script>
  <!-- endinject -->
```